File: src/com/tal/android/plugin/utils/ViewUtils.java
package com.tal.android.plugin.utils;

import javax.swing.JComponent;

public final class ViewUtils {
    public static void hide(JComponent... views) {
        if (ListUtils.isEmpty(views)) {
            return;
        }
        for (JComponent view : views) {
            view.setVisible(false);
        }
    }

    public static void show(JComponent... views) {
        if (ListUtils.isEmpty(views)) {
            return;
        }
        for (JComponent view : views) {
            view.setVisible(true);
        }
    }
}

File: src/com/nerdscorner/android/plugin/utils/ViewUtils.kt
package com.nerdscorner.android.plugin.utils

import javax.swing.JComponent

object ViewUtils {
    fun hide(vararg views: JComponent) {
        if (views.isNullOrEmpty()) {
            return
        }
        for (view in views) {
            view.isVisible = false
        }
    }

    fun show(vararg views: JComponent) {
        if (views.isNullOrEmpty()) {
            return
        }
        for (view in views) {
            view.isVisible = true
        }
    }
}
